ci_test.sh

```diff
@@ -12,7 +12,17 @@ if [ ! -f _output/wc/KWC${KVER}.sys ] ; then
   exit 1
 fi
 
-echo GCC and Watcom kernels have all been built
+if [ ! -f _output/wc_dos/KWC38632.sys ] ; then
+  echo Watcom DOS built kernel not present
+  exit 1
+fi
+
+if [ ! -f _output/tc_dos/KTC38632.sys ] && [ -d ${HOME}/.dosemu/drive_c/tc201 ] ; then
+  echo Turbo C 2.01 built kernel not present
+  exit 1
+fi
+
+echo Kernels have all been built
 find _output -ls
 
 cd test
```

Comment:
Hi @ecm-pushbx I'd like to add to your boot test here. The DOS built kernels are built for 386, but they don't have to be I just thought I'd add some variety. Is there any special command line required for them? BTW there is no Turbo C version at present, as it fails to build `sys`, `kernel` and maybe more.

Reply:
> Is there any special command line required for them?

Just call the test script with `./test.sh ../_output/PATH/FILE.sys diskID bootID 'boot ID: '` after chdir to the repo's test/ directory, where PATH and FILE point to your kernel and ID is something unique, ideally with 4 or less letters. (I'd suggest wcd for Watcom DOS and tcd for Turbo C DOS.) It's called like that here: https://github.com/FDOS/kernel/blob/edd073243393f65070fe304d91346b8ce0ee313f/ci_test.sh#L19

The three parameters to test.sh are called:

```
KERNELPATHNAME="$1"
DISKETTE="$2"
BOOT="$3"
testrunname="$4"
```

Quoted from https://github.com/FDOS/kernel/blob/edd073243393f65070fe304d91346b8ce0ee313f/test/test.sh#L74